src/main/java/com/yahoo/memory/AllocateDirectMap.java

```diff
@@ -64,10 +61,6 @@
 
   static {
```

Comment:
Could put static initializer before instance fields?

Reply:
We refer to them outside the static {...}.

Reply:
I don't understand this comment

Reply:
> Could put static initializer before instance fields?

Then I don't understand what you are trying to do.

Reply:
It's surprising to see non-static instance fields going before a long static block. When I looked at the constructor, I couldn't find field declarations just above it, and I didn't think about looking above the static block, because didn't expect to find instance fields there.

Reply:
Ahh. I'll just move the static block to the top!

Reply:
I'll check all this in to the branch so you can double-check.